app-backend/db/src/main/resources/migrations/V5__Task_and_Task_Actions.sql

```diff
@@ -0,0 +1,33 @@
+-- Create tasks table
+
+CREATE TYPE task_status AS ENUM (
+    'UNLABELED', 'LABELING_IN_PROGRESS', 'LABELED', 'VALIDATION_IN_PROGRESS', 'VALIDATED'
+);
+
+CREATE TABLE tasks (
+  id uuid primary key,
+  created_at timestamp without time zone  not null,
+  created_by text not null references users (id),
+  modified_at timestamp without time zone not null,
+  modified_by text not null,
+  project_id uuid not null,
```

Comment:
So it looks like from the spec that there needs to be an `owner` field: https://github.com/raster-foundry/raster-foundry/blob/develop/docs/swagger/spec.yml#L7785

Reply:
Ah I didn't notice good catch